deploy/docker-compose.yml

```diff
@@ -169,4 +169,4 @@ services:
         condition: service_healthy
 
 volumes:
-  postgres_data:
+  postgres_data:
```

Comment:
new line

Reply:
I restored this to match the main branch on upstream

Reply:
wdym? Its showing you removing a newline